modules/installation-initializing-manual.adoc

```diff
@@ -33,13 +47,19 @@ names for the installation assets might change between releases. Use caution
 when copying installation files from an earlier {product-title} version.
 ====
 
-. Customize the following `install-config.yaml` file template and save 
+. Customize the following `install-config.yaml` file template and save
 it in the `<installation_directory>`.
 +
 [NOTE]
 ====
 You must name this configuration file `install-config.yaml`.
 ====
+ifdef::disconnected[]
+** You must provide the contents of the certificate for your mirror repository
+in the `additionalTrustBundle` section.
```

Comment:
Only if your mirror certificate is not signed by a CA RHCOS trusts by default.  Maybe weaken the "must"?

Reply:
@wking, can you tell me a little more about how this workflow would work? I'm happy to soften that requirement, but I'd need to add some notes in other topics. (Like in the topic about setting up a mirror repo and probably elsewhere in the installation docs.)

Reply:
If you're mirroring to a registry whose cert RHCOS trusts by default (e.g. from quay.io to docker.io), you don't need to add additional trust for your mirror registry.

Reply:
I'm adding that exception.